huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_instance_test.go

```diff
@@ -553,20 +545,35 @@ func testAccCheckRdsInstanceExists(name string, instance *instances.RdsInstanceR
 	}
 }
 
+func testAccRdsInstance_base() string {
```

Comment:
It is better to pass the name through the input parameter. Avoid errors caused by duplicate names.